docs/connector-development/README.md

```diff
@@ -124,8 +124,8 @@ Once you've finished iterating on the changes to a connector as specified in its
    /publish connector=(connectors|bases)/<connector_name>
    ```
    
-4. OPTIONAL: Only necessary if the automated connector version bump fails
-   * Update the connector definition in the Airbyte connector index to use the new version:
+4. OPTIONAL: Necessary if this is a new connector, or the automated connector version bump fails
```

Comment:
if the bump fails, that fails the entire `/publish` command right? I.e. you'd have to rerun `/publish`, at which point it would try to bump again?

Reply:
I kept the flow the same order as before, so first it publishes the connector and if that succeeds, it then does the auto version bump. 

That means it can successfully publish but then fail auto-bump, this _shouldn't_ happen so if it does, manual eyes figuring out why is a good thing. This should be clear based on the comments that get added back to the git comment.

Reply:
ah nice, makes sense :+1: